typescript/sdk/src/contracts/contracts.ts

```diff
@@ -148,10 +150,15 @@ export function connectContracts<F extends HyperlaneFactories>(
   contracts: HyperlaneContracts<F>,
   connection: Connection,
 ): HyperlaneContracts<F> {
-  return objMap(
-    contracts,
-    (_, contract) => contract.connect(connection) as typeof contract,
-  );
+  const connectedContracts = objMap(contracts, (_, contract) => {
+    if (!contract.connect) {
+      return undefined;
+    }
```

Comment:
what case does this handle?

Reply:
this is the case when the entry is not a contract, but a testnet entry with ISMs. I assumed the ism contracts shouldn't be `connect`ed (not sure what this does), so I'm returning `undefined` and filtering out undefined entries below